### PR TITLE
[10.0][MIG] barcodes_generator_partner

### PR DESCRIPTION
**Pull request by lasley**
Migration of `barcodes_generator_partner` to v10.

To be honest, curiosity got the best of me on this one. I don't need this module, but as mentioned in https://github.com/OCA/stock-logistics-barcode/pull/65#issuecomment-269546792 I think that a lot of the coverage gap is filled by having both modules. This is to either prove of disprove my hypothesis.
	
* Rename manifest
* Change openerp references to odoo
* Bump version

Depends:
- [x] #65

### CODE REVIEW

**Comment by lasley** [comment]
Awwww yehhhhh hypothesis proven!

Awyeh

File: barcodes_generator_abstract/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+{
+    'name': 'Generate Barcodes (Abstract)',
+    'summary': 'Generate Barcodes for Any Models',
+    'version': '10.0.1.0.0',
+    'category': 'Tools',
+    'author':
+        'GRAP,'
+        'La Louve,'
+        'Odoo Community Association (OCA)',
+    'website': 'http://www.odoo-community.org',
```
Comment (lasley) [inline barcodes_generator_abstract/__manifest__.py]: https

File: barcodes_generator_partner/__manifest__.py [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+{
+    'name': 'Generate Barcodes for Partners',
+    'summary': 'Generate Barcodes for Partners',
+    'version': '10.0.1.0.0',
+    'category': 'Tools',
+    'author':
+        'GRAP,'
+        'La Louve,'
+        'Odoo Community Association (OCA)',
+    'website': 'http://www.odoo-community.org',
```
Comment (lasley) [inline barcodes_generator_partner/__manifest__.py]: https

**Comment by lasley** [comment]
Readded the old commits that I somehow removed. Good for review!

**Comment by legalsylvain** [comment]
Thanks ! :+1:

For other reviewer, please review only the last commit https://github.com/OCA/stock-logistics-barcode/pull/67/commits/cae850cef2abb97cc4a0d5410ece76a2a8d319a9
(more trivial to review).

**Review by legalsylvain — APPROVED** [review]

**Comment by legalsylvain** [comment]
I propose to merge this PR (optimistic merge) : 
-> changes are trivial (ref : last commits)
-> it increases code coverage of the abstract module

An external PoV is welcome. CC @OCA/logistics-maintainers 

kind regards.

**Review by gurneyalex — APPROVED** [review]

**Pull request merged**